BlockSettleUILib/LoginWindow.cpp

```diff
@@ -35,16 +35,14 @@ namespace {
 }
 
 LoginWindow::LoginWindow(const std::shared_ptr<spdlog::logger> &logger
+   , BsClient *bsClient
```

Comment:
Why can't we use BsClient as a shared_ptr instead? This will simplify such things a lot.

Reply:
yes, sure, but I don't see any difficulties here that require `shared_ptr`